src/components/fabSpeedDial/fabSpeedDial.spec.js

```diff
@@ -0,0 +1,95 @@
+describe('<md-fab-speed-dial> directive', function() {
+
+  beforeEach(module('material.components.fabSpeedDial'));
+
+  function setup(template) {
+    var el;
+
+    inject(function($compile, $rootScope) {
+      el = $compile(template)($rootScope.$new());
+      $rootScope.$apply();
+    });
```

Comment:
Won't `pageScope` be the same as `scope` in the above snippet ?
Did you by any chance mean `scope = element.isolateScope()` ?
<sub>(I haven't looked at the actual code; I'm just making assumptions here (and they might be totally off).</sub>